packages/discord.js/src/managers/VoiceStateManager.js

```diff
@@ -32,6 +33,27 @@ class VoiceStateManager extends CachedManager {
     if (cache) this.cache.set(data.user_id, entry);
     return entry;
   }
+
+  /**
+   * Obtains a user's voice state from discord or from the cache if it's already available.
+   * @param {GuildMemberResolvable} member The member whose voice state is to be fetched
+   * @param {BaseFetchOptions} [options] Additional options for this fetch
+   * @returns {Promise<VoiceState>}
+   * @example
+   * // Fetch a member's voice state
+   * guild.voiceStates.fetch("851588007697580033")
```

Comment:
The docs uses hydrabolt's id
```suggestion
   * guild.voiceStates.fetch("66564597481480192")
```